web/src/integration-test/java/ca/hendriks/bartender/web/functionaltest/ingredient/BddIngredientService.java

```diff
@@ -56,15 +57,22 @@ public Ingredient findIngredient(final String ingredientName) {
         return ingredientRepository.findByName(ingredientName);
     }
 
-    public Ingredient updateIngredient(final String ingredientName, final IngredientType ingredientType){
-        final int ingredientFromRepoId = findIngredient(ingredientName).getId();
-        final Ingredient ingredientToUpdate = new Ingredient(ingredientFromRepoId, ingredientName, ingredientType);
-        ingredientRepository.save(ingredientToUpdate);
-        return ingredientToUpdate;
+    public void updateIngredient(final String originalIngredientName, final String originalIngredientType,
+                                       final String updatedIngredientName, final String updatedIngredientType){
+        List<String> ingredientUpdates = new ArrayList<>();
+        ingredientUpdates.add(originalIngredientName);
+        ingredientUpdates.add(originalIngredientType);
+        ingredientUpdates.add(updatedIngredientName);
+        ingredientUpdates.add(updatedIngredientType);
+        mockMvc.post("/ingredients/update", ingredientUpdates);
```

Comment:
You shouldn't use `POST` to update a resource:
https://stackoverflow.com/questions/630453/what-is-the-difference-between-post-and-put-in-http

Reply:
Oops yes, didn't see that I have control over BddMockMvcService